razzle.config.js

```diff
@@ -93,6 +93,9 @@ const defaultModify = ({
       ? 'static/js/[name].js'
       : 'static/js/[name].[chunkhash:8].js';
 
+    const TerserPlugin = require('terser-webpack-plugin');
+    const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
```

Comment:
These are probably better if moved to the top of the file for code quality and readability

Reply:
ups, true!

Reply:
done.